test/server_test.lua

```diff
@@ -322,11 +322,9 @@ g.test_wait_when_server_is_not_running_by_bad_option = function()
     t.assert_equals(status, false)
     t.assert_str_contains(msg, expected_msg)
     t.assert_equals(s1.process:is_alive(), false)
-    s1:clean()
```

Comment:
Please put the `Not fail server:drop() if the process is already dead` commit before this one and change it to `:drop()`.

Reply:
Commits have been swapped.